grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1196,9 +1200,6 @@ class StockMovementService {
         shipment.destination = stockMovement.destination
         shipment.description = stockMovement.description
 
-        // These values need defaults since they are not set until step 6
-        shipment.expectedShippingDate = new Date()
```

Comment:
I think you can leave it here and just update this date when the requisition state is changed when going to the last page